JasmineTests/MutableCollection+ExtensionsTests.swift

```diff
@@ -0,0 +1,19 @@
+import XCTest
+@testable import Jasmine
+
+class MutableCollectionExtensionsTests: XCTestCase {
+
+    private let iterations = 10
+
+    func testShuffle() {
+        let collection = [1, 2, 3]
+        var lastElementMoved = false
+        for _ in 0...iterations {
+            let shuffled = collection.shuffled()
+            lastElementMoved = lastElementMoved || collection[2] == shuffled[2]
+            XCTAssertEqual(collection, shuffled.sorted(), "Elements were changed")
+        }
+        XCTAssertTrue(lastElementMoved, "Last element was not swapped")
```

Comment:
Should unit test for random be done in this manner? The test is "theoretically" unsound as it has a minute chance to fail even when the function is correctly implemented.
http://softwareengineering.stackexchange.com/questions/147134/how-should-i-test-randomness